code/modules/atmospherics/machinery/components/unary_devices/airlock_pump.dm

```diff
@@ -342,6 +344,7 @@
 	if(!on)
 		return FALSE
 	set_on(FALSE)
+	update_use_power(IDLE_POWER_USE)
```

Comment:
Should this just be updated in `set_on`?

Reply:
why not